invenio_rdm_records/resources/serializers/datacite/schema.py

```diff
@@ -177,10 +180,11 @@ class DataCite43Schema(Schema):
     def get_type(self, obj):
         """Get resource type."""
         resource_type = obj["metadata"]["resource_type"]
-
+        vocabulary = Vocabularies.get_vocabulary('resource_type')
+        entry = vocabulary.get_entry_by_dict(resource_type)
         return {
-            'resourceTypeGeneral': "FIXME",
-            'resourceType': "FIXME",
+            'resourceTypeGeneral': entry.get("datacite_general", "Other"),
```

Comment:
minor: do we want to default to `Other`? Or fail if not found? To avoid pushing "wrong" data or from a not properly configured instance (e.g. *you just migrated and misconfigured, new records will be always other*)

Reply:
Let's go with ``Other`` to ensure it's registered. Otherwise we have to do a lot of error handling and have ways to fix it easily, and we don't,.